frontend/src/components/c-ramp.vue

```diff
@@ -211,6 +223,10 @@ export default {
     &--color4 {
       border-bottom: $height rgba(mui-color('pink'), .5) solid;
     }
+
+    &--color-deletes {
+      border-bottom: $height rgba(mui-color('red'), .5) solid;
```

Comment:
I think the colors of certain pure deletion commit ramps are still generated from date. The ramp of commit `96d61c9` in the screenshot is colored blue here

![image](https://user-images.githubusercontent.com/54735123/216952726-c290ad23-4343-4d60-b760-b9b9f54785f6.png)

Reply:
Yes, thanks for pointing that out!
I have fixed the issue, and now the thin red ramps appear for both `commit` granularity, and `day`/`week` granularity.

### `commit` granularity:
<img width="1440" alt="Screenshot 2023-02-07 at 12 12 00 AM" src="https://user-images.githubusercontent.com/68203159/217024296-fae2eead-0190-4a0d-9bcc-2b82f4db35c0.png">

### `day` granularity:
<img width="1440" alt="Screenshot 2023-02-07 at 12 17 58 AM" src="https://user-images.githubusercontent.com/68203159/217025660-ced5a9d0-59bb-40c9-b9c8-14c83d6017fe.png">